pyqtgraph/graphicsItems/PlotCurveItem.py

```diff
@@ -496,7 +515,10 @@ def updateData(self, *args, **kargs):
         if self.opts['stepMode'] in ("center", True):  ## check against True for backwards compatibility
             if self.opts['stepMode'] is True:
                 import warnings
```

Comment:
I know you didn't just add this, but would you mind removing this import here given the static code checker is complaining about it?